### PR TITLE
Added lightware lidar sf45 rotating lidar doc

### PR DESCRIPTION
**Pull request by dirksavage88**
Setup information for the lightware sf45 rotating lidar introduced in 1.14. 

Basic setup steps to include an asset showing the obstacle distance overlay in QGC

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thank you @dirksavage88 - I'm just back from PX4 dev days. Will look at this and your other updates next week. Not ignoring!

File: en/computer_vision/collision_prevention.md [hunk 1]
```diff
@@ -130,16 +127,49 @@ If the vehicle feels 'stuck' with only a single distance sensor pointing forward
...
+
+The SF45 must be connected via a UART/serial port.
+
+[LightWare Studio](https://www.lightwarelidar.com/resources-software) configuration:
+
+- In the LightWare Studio app enable scanning, set the scan angle, and change the baud rate to `921600`.
+
+PX4 Configuration:
+
+- Add the [lightware_sf45_serial](../modules/modules_driver_distance_sensor.html#lightware-sf45-serial) driver in [menuconfig](../hardware/porting_guide_config.md#px4-menuconfig-setup):
```
Comment (hamishwillee) [inline en/computer_vision/collision_prevention.md:149]:
```suggestion
- Add the [lightware_sf45_serial](../modules/modules_driver_distance_sensor.md#lightware-sf45-serial) driver in [menuconfig](../hardware/porting_guide_config.md#px4-menuconfig-setup):
```

File: en/computer_vision/collision_prevention.md [hunk 2]
```diff
@@ -130,16 +127,49 @@ If the vehicle feels 'stuck' with only a single distance sensor pointing forward
...
 At time of writing PX4 allows you to use the [Lanbao PSK-CM8JL65-CC5](../sensor/cm8jl65_ir_distance_sensor.md) IR distance sensor for collision prevention "out of the box", with minimal additional configuration:
+
 - First [attach and configure the sensor](../sensor/cm8jl65_ir_distance_sensor.md), and enable collision prevention (as described above, using [CP_DIST](#CP_DIST)).
 - Set the sensor orientation using [SENS_CM8JL65_R_0](../advanced_config/parameter_reference.md#SENS_CM8JL65_R_0).
 
+### LightWare LiDAR SF45 Rotating Lidar
+
+PX4 v1.14 (and later) supports the [LightWare LiDAR SF45](https://www.lightwarelidar.com/shop/sf45-b-50-m/) rotating lidar which provides 320 degree sensing.
+
+The SF45 must be connected via a UART/serial port.
```
Comment (hamishwillee) [inline en/computer_vision/collision_prevention.md:141]:
```suggestion
The SF45 must be connected via a UART/serial port and configured as described below (In addition to the [collision prevention setup](#px4-software-setup)).
```

File: en/computer_vision/collision_prevention.md [hunk 3]
```diff
@@ -130,16 +127,49 @@ If the vehicle feels 'stuck' with only a single distance sensor pointing forward
...
+    Make sure GPS or Telemetry are not enabled on this port.
+  - [SF45_ORIENT_CFG](../advanced_config/parameter_reference.md#SF45_ORIENT_CFG): Set the orientation of the sensor (facing up or down)
+  - [SF45_UPDATE_CFG](../advanced_config/parameter_reference.md#SF45_UPDATE_CFG): Set the update rate
+  - [SF45_YAW_CFG](../advanced_config/parameter_reference.md#SF45_YAW_CFG): Set the yaw orientation
+
+In QGroundControl you should see an [OBSTACLE_DISTANCE](https://mavlink.io/en/messages/common.html#OBSTACLE_DISTANCE) message in the [MAVLink console](../debug/mavlink_shell.md#qgroundcontrol-mavlink-console) if collision prevention is configured correctly and active.
+
+The obstacle overlay in QGC will look like this:
+
+![sf45](../../assets/sf45/sf45_obstacle_map.png)
```
Comment (hamishwillee) [inline en/computer_vision/collision_prevention.md:163]: FYI, I've modified this to better cross link to other relevant information that might not be so well understood by some users - such as menuconfig. Also to make it a little more readable. The core of it is the same.

File: en/sensor/rangefinders.md [hunk 4]
```diff
@@ -42,6 +42,8 @@ The rangefinders are enabled using the parameter [SENS_EN_MB12XX](../advanced_co
 PX4 supports: SF11/c and SF/LW20.
 PX4 can also be used with the following discontinued models: SF02, SF10/a, SF10/b, SF10/c.
 
+PX4 also supports the [LightWare LiDAR SF45 Rotating Lidar](https://www.lightwarelidar.com/shop/sf45-b-50-m/) for [collision prevention](../computer_vision/collision_prevention.md#lightware-lidar-sf45-rotating-lidar) applications.
```
Comment (hamishwillee) [inline en/sensor/rangefinders.md:45]: @dirksavage88 I also added a link to the lidar and our docs here. Would this lidar ever be used for "general use" or is it really just for the collision prevention/obstacle avoidance use case?

What I'm thinking is whether I need to have a peer of https://docs.px4.io/main/en/sensor/rangefinders.html for scanning rangefinders or similar and move some of the setup into that. Just a thought - won't block merging this.

File: en/computer_vision/collision_prevention.md (hunk 3 above)
Comment (hamishwillee) [reply en/computer_vision/collision_prevention.md:163]: PS But a quick scan to check you are still happy with this would be good!

Comment (dirksavage88) [reply en/computer_vision/collision_prevention.md:163]: Looks good, thanks

File: en/sensor/rangefinders.md (hunk 4 above)
Comment (dirksavage88) [reply en/sensor/rangefinders.md:45]: It’s only for collision prevention. There is a ROS driver out there but for the px4 avoidance package it probably won’t work out of the box

**Pull request merged**